src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Update/MappingProfile.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using AutoMapper;
 using Digdir.Domain.Dialogporten.Application.Common.Extensions;
-using Digdir.Domain.Dialogporten.Application.Features.V1.Dialogs.Queries.EndUser.Get;
+using Digdir.Domain.Dialogporten.Application.Features.V1.EndUser.Dialogs.Queries.Get;
```

Comment:
Note to self: Her er det en bug - ServiceOwner update skal ikke referere til EndUser Get.

Reply:
Dette er vel pga den her:

```
        // To support json patch
        CreateMap<GetDialogDto, UpdateDialogDto>();
        CreateMap<GetDialogDialogActivityDto, UpdateDialogDialogActivityDto>();
        CreateMap<GetDialogDialogApiActionDto, UpdateDialogDialogApiActionDto>();
        CreateMap<GetDialogDialogApiActionEndpointDto, UpdateDialogDialogApiActionEndpointDto>();
        CreateMap<GetDialogDialogGuiActionDto, UpdateDialogDialogGuiActionDto>();
        CreateMap<GetDialogDialogElementDto, UpdateDialogDialogElementDto>();
        CreateMap<GetDialogDialogElementUrlDto, UpdateDialogDialogElementUrlDto>();
 ```